cypress/e2e/2-advanced-examples/bookStore.cy.js

```diff
@@ -0,0 +1,27 @@
+
+import { Login } from "../pages/BookStore";
+import { Profile } from "../pages/Profile";
+
+describe('Adding Book', () => {
+    beforeEach (() => {
+    cy.fixture("account-bookpage").as("account");
+    cy.visit('https://demoqa.com/books');
+
+});
+it("Add Book Sucessfully", () => {
+    cy.get('@account').then((account) => {
+    Login
+    .loginBookStore()
+    .typeUserName(account.username)
+    .typePassword(account.password)
+    .clickLogin()
+    .clickToSeeBookInfor()
+    .clickAddBook()
+    Profile
+    .clickToProfile ()
+    .checkUserNamevalue(account.username)
```

Comment:
No need to check if username value is displayed correctly, in this flow what you need is verifying if book is add into collection or not